src/NumFieldOrd/NfOrd/LLL.jl

```diff
@@ -125,7 +125,7 @@ function _lll(A::AbsNumFieldOrderIdeal{AbsSimpleNumField, AbsSimpleNumFieldElem}
 
 
     round_scale!(d, c, prec)
-    ccall((:fmpz_mat_mul, libflint), Nothing, (Ref{ZZMatrix}, Ref{ZZMatrix},  Ref{ZZMatrix}), g, (b.num), d)
+    d = mul!(g, b.num, d)
```

Comment:
```suggestion
    g = mul!(g, b.num, d)
```

Reply:
sneaky bug

Reply:
Argh, sorry & thanks for catching it

Reply:
first time I am happy that we are always running the book tests